c10/xpu/impl/xpu_cmake_macros.h.in

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+// Automatically generated header file for the C10 XPU library.  Do not
+// include this file directly. Instead, include c10/xpu/XPUMacros.h
+
+#cmakedefine C10_XPU_BUILD_SHARED_LIBS
```

Comment:
Please delete, if CUDA backend has something this does not look like a good reason to copy the same pattern. 
If it is really needed, it sounds like a better idea to refactor the common logic to some generic file (to be frank, I'm not sure why CUDA variant was introduced, when `C10_BUILD_SHARED_LIBS` already exists

Reply:
This is used for windows build, let me remove it.